code/modules/research/xenobiology/crossbreeding/loyal.dm

```diff
@@ -0,0 +1,745 @@
+	. = ..()
+	if(!proximity || !istype(target, /obj/item) || target.GetComponent(/datum/component/loyal_effect))
+		return FALSE
+	var/datum/component/loyal_effect/new_effect = target.AddComponent(loyaleffect, target)
+	to_chat(user, "<span class='notice'>You smear the [target] with [src], making it [new_effect.prefix]!</span>")
+	qdel(src)
+	return TRUE
+
+/datum/component/loyal_effect
+	var/prefix = "weird"
+	var/obj/item/attached
+
+/datum/component/loyal_effect/Initialize(obj/item/attached_item)
+	if(!attached_item)
+		return
+	attached = attached_item
+	attached.name = "[prefix] [attached.name]"
+
+/obj/item/slimecross/loyal/grey
+	colour = "grey"
+	effect_desc = "Makes an item very sticky."
+	loyaleffect = /datum/component/loyal_effect/grey
+
+/obj/item/slimecross/loyal/grey/afterattack(obj/item/target,mob/user,proximity)
+	if(HAS_TRAIT_FROM(target, TRAIT_NODROP, GLUED_ITEM_TRAIT))
+		to_chat(user, "<span class='warning'>[target] is already sticky!</span>")
+		return
+	if(!..()) //Is this shitcode?
+		return
+	ADD_TRAIT(target, TRAIT_NODROP, GLUED_ITEM_TRAIT) //Pretty much just syndie glue
+	target.desc += " It looks sticky."
+
+/datum/component/loyal_effect/grey
+	prefix = "sticky"
+
+/obj/item/slimecross/loyal/orange
+	colour = "orange"
+	effect_desc = "Makes an item grow nettles that burn the hands of anyone who picks it up without gloves."
+	loyaleffect = /datum/component/loyal_effect/orange
+
+/datum/component/loyal_effect/orange
+	prefix = "nettling"
+
+/datum/component/loyal_effect/orange/Initialize(obj/item/attached_item)
+	. = ..()
+	RegisterSignal(attached_item, COMSIG_ITEM_PICKUP, .proc/pickup_burn)
+
+/datum/component/loyal_effect/orange/proc/pickup_burn(datum/source, mob/user)
+	if(!iscarbon(user))
+		return FALSE
+	var/mob/living/carbon/C = user
+	if(C.gloves)
+		return
+	if(HAS_TRAIT(C, TRAIT_PIERCEIMMUNE))
+		return
+	var/hit_zone = (C.held_index_to_dir(C.active_hand_index) == "l" ? "l_":"r_") + "arm"
+	var/obj/item/bodypart/affecting = C.get_bodypart(hit_zone)
+	if(affecting)
+		if(affecting.receive_damage(0, 10))
+			C.update_damage_overlays()
+	to_chat(C, "<span class='userdanger'>[attached] burns your bare hand!</span>")
+
+/obj/item/slimecross/loyal/purple
+	colour = "purple"
+	effect_desc = "Makes a piece of clothing heal whatever body part it is worn over."
+	loyaleffect = /datum/component/loyal_effect/purple
+
+/datum/component/loyal_effect/purple
+	prefix = "medicinal"
+	var/mob/living/carbon/attached_user
+
+/datum/component/loyal_effect/purple/Initialize(obj/item/attached_item)
+	. = ..()
+	if(istype(attached_item, /obj/item/clothing)) //We only care if it's clothes.
+		START_PROCESSING(SSobj,src)
+		RegisterSignal(attached_item, list(COMSIG_ITEM_EQUIPPED, COMSIG_ITEM_DROPPED), .proc/equippedChanged)
+
+/datum/component/loyal_effect/purple/Destroy()
+	STOP_PROCESSING(SSobj,src)
+	return ..()
+
+/datum/component/loyal_effect/purple/proc/equippedChanged(datum/source, mob/living/carbon/user, slot)
```

Comment:
If loyal purple is used on clothes while worn, it doesn't start processing until they are taken off and put back on. I gotta change that.